### ISSUE TITLE
Unable to fetch ReactiveCocoa

### ISSUE THREAD

**Issue by lynxluna** [Carthage/Carthage#823]
I tried to fetch ReactiveCocoa with freshly installed Xcode7 and only intend for testing, so there's no signing identity (because Xcode7 allows it).

I got this error:

```
$ carthage update --platform iOS
*** Fetching ReactiveCocoa
*** Fetching Result
*** Downloading ReactiveCocoa.framework binary at "v4.0-alpha.1"
*** Checking out Result at "76f9a972ed61ff872b731460bc610c8acbfae58c"
*** xcodebuild output can be found in /var/folders/w0/_wgjy1r56kd7sxrbxp26hr1w0000gn/T/carthage-xcodebuild.zN7FhN.log
*** Building scheme "Result-iOS" in Result.xcodeproj
*** WARNING: Skipping build for "iphonesimulator" SDK because the necessary signing identity "iPhone Developer" is not installed
*** WARNING: Skipping build for "iphoneos" SDK because the necessary signing identity "iPhone Developer" is not installed
fatal error: No valid SDKs found to build (you're missing one or more signing identities): file /Users/mdiep/Repositories/Carthage/Carthage/Source/CarthageKit/Xcode.swift, line 898
```

I'm using 0.9.2. I believe this is related to #789. And why does it need signing on the `iphonesimulator` SDK on Result project?

**Comment by mdiep** [comment]
The Result project must be configured to code sign for the simulator.

**Comment by lynxluna** [comment]
So Carthage still requires signing for `iphoneos` target even if Xcode 7 allows us to just run the code on the device without signing?

**Comment by mdiep** [comment]
> even if Xcode 7 allows us to just run the code on the device without signing?

I don't think that's true. AFAIK, you still need to code sign an iOS framework in order to build it.

**Comment by ikesyo** [comment]
This can be closed since #947 was merged and released.
